drake/systems/plants/test/testAccelerometer.cpp

```diff
@@ -1,78 +1,105 @@
 #include "drake/systems/plants/RigidBodySystem.h"
 #include "drake/util/testUtil.h"
 
-using namespace std;
-using namespace Drake;
-using namespace Eigen;
-
-
-Vector3d getAccelerometerOutput(shared_ptr<RigidBodySystem> const& sys, Vector3d const& rpy, Vector4d const& u) {
+#include "drake/util/eigen_matrix_compare.h"
+#include "gtest/gtest.h"
+
+using Eigen::Vector3d;
+using Eigen::Vector4d;
+using Eigen::VectorXd;
+using std::shared_ptr;
+using std::make_shared;
+using Drake::getDrakePath;
+using Drake::RigidBodySystem;
+using drake::util::MatrixCompareType;
+using Drake::RigidBodyAccelerometer;
+
+namespace drake {
+namespace test {
```

Comment:
Oh, I see (belatedly): you're using the `test` namespace to protect the production namespaces from pollution.  Great idea, but why not just use an anonymous namespace?